docs/usage.md

```diff
@@ -329,14 +329,21 @@ Workflows allow you to filter by:
 
 ### Workflow Actions
 
-There is currently one type of workflow action, "Assignment", which can assign:
+There are currently two types of workflow actions, "Assignment", which can assign:
 
 - Title, see [title placeholders](usage.md#title-placeholders) below
 - Tags, correspondent, document type and storage path
 - Document owner
 - View and / or edit permissions to users or groups
 - Custom fields. Note that no value for the field will be set
```

Comment:
Just while we're in here, this can do storage paths as well

Reply:
I think its already in the list, unless I misunderstood

Reply:
Nope, I was just blind, missed it on the first bullet